src/mindist.rs

```diff
@@ -149,7 +149,7 @@ fn S(u: f64, v: f64, bez1: &[Vec2], bez2: &[Vec2]) -> f64 {
     let m = bez2.len() - 1;
```

Comment:
Just out of curiousity, is it possible for this to receive 0-length inputs?

Reply:
Nope, because it's called with the results of `as_vec2_vec`. But some asserts wouldn't be bad.